Library/Homebrew/extend/os/linux/system_config.rb

```diff
@@ -40,10 +40,20 @@ def host_ruby_version
       out
     end
 
+    def wsl_version(kernel)
+      return "no" unless /-microsoft/i.match?(kernel)
```

Comment:
```suggestion
      return unless /-microsoft/i.match?(kernel)
```

Let's not print anything here on the typical, non-WSL case.